reagurk/src/components/footer.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+
+export default function Footer(): JSX.Element {
+
+  return (
+    <>
+      <footer className="text-gray-400 bg-gray-800 body-font fixed
+             inset-x-0
+             bottom-0">
+        <div className="container px-5 py-8 mx-auto flex items-center sm:flex-row flex-col">
+          <a className="flex title-font font-medium items-center md:justify-start justify-center text-white">
+            <img src="https://cdn.discordapp.com/emojis/792081164466126868.png?size=56" alt="gurkult logo" />
+            <span className="ml-3 text-xl">Gurkult</span>
+          </a>
+          <p className="text-sm text-gray-400 sm:ml-4 sm:pl-4 sm:border-l-2 sm:border-gray-800 sm:py-2 sm:mt-0 mt-4">© 2021 Gurkult 
+           
+          </p>
+          <span className="inline-flex sm:ml-auto sm:mt-0 mt-4 justify-center sm:justify-start">
+            <a className="text-gray-400"
+              href='https://github.com/gurkult'>

```

Comment:
This should use a constant

Reply:
What? I don't see which constant are you talking about.

Reply:
> What? I don't see which constant are you talking about.

https://github.com/gurkult/reagurk/blob/bc256c14d1ff300d316f7b9b8ec70ba3e468fd97/reagurk/src/constants.ts#L1

https://github.com/gurkult/reagurk/blob/bc256c14d1ff300d316f7b9b8ec70ba3e468fd97/reagurk/src/constants.ts#L3